### ISSUE TITLE
Vidoe is not showing, but audio is playing

### ISSUE THREAD

**Issue by Alpesh-Android** [PierfrancescoSoffritti/android-youtube-player#621]
Hello sir,

**Comment by romanandroiddev** [comment]
Hello, can you explain shortly, how you solved this problem?

**Comment by PierfrancescoSoffritti** [comment]
Are you using a device or an emulator? this is a common issue on emulators.

**Comment by romanandroiddev** [comment]
Yeah, it is android emulator, but I found out that, in real device it is
working.

пн, 6 мар. 2023 г., 20:42 Pierfrancesco Soffritti ***@***.***
